src/app/routes/api/platform/v1/admin/forms/admin-forms.platform.routes.ts

```diff
@@ -0,0 +1,53 @@
+import { Router } from 'express'
+
+import { rateLimitConfig } from '../../../../../../config/config'
+import { authenticateApiKeyPlatformUser } from '../../../../../../modules/auth/auth.middlewares'
+import * as AdminFormController from '../../../../../../modules/form/admin-form/admin-form.controller'
+import { limitRate } from '../../../../../../utils/limit-rate'
+
+export const AdminFormsPlatformRouter = Router()
+
+// All routes in this handler should be protected by authentication.
+AdminFormsPlatformRouter.use(authenticateApiKeyPlatformUser)
+
+AdminFormsPlatformRouter.route('/:formId([a-fA-F0-9]{24})/webhooksettings')
+  /**
+   * Retrieve the webhook and response mode settings of the specified form
+   * @security bearer
+   * @route GET /admin/forms/:formId/settings
+   * @param body the user email
+   *
+   * @returns 200 with latest form settings
+   * @returns 401 when current user is not logged in
+   * @returns 403 when current user does not have permissions to obtain form settings
```

Comment:
Qn - when would these error codes apply for a platform user?

Reply:
^ only the 401, actually. I realised 403 applies if the user listed in the userEmail doesn't have write permissions for that specific form

Reply:
thanks @tshuli ! I've refined the description in [9cdd65b](https://github.com/opengovsg/FormSG/pull/6608/commits/9cdd65b29676ff8e77cd2edbe08c7d36c5b3af03), 401 is returned when the user doesn't provide a valid API key, or is not a platform user